### DISCUSSION TITLE
What about add this project to Void?

### DISCUSSION THREAD

**Discussion by LambLeeg** [deepjyoti30/ytmdl#148] [General]
So, Void is popular distro which is about between Gentoo and Arch (I mean by difficulty level) and would be nice to add it to xbps-src that is like AUR that developers support already.

**Comment by deepjyoti30** [comment]
@LambLeeg I would like to add it to the sources, however I cannot test it on my machine before publishing it, so if you wanna take a dig, you're more than welcome. Or if anyone else does considers adding it, I would totally consider adding it to the README.
